packages/api/tests/lambdas/test-kinesis-consumer.js

```diff
@@ -127,8 +123,6 @@ test.before(async () => {
     payload: get(item, 'payload', {}),
     definition: workflowDefinition,
   }));
-  sandbox.stub(Provider.prototype, 'get').callsFake((providerArg) => providerArg);
```

Comment:
Surprised that removing this stub works because I didn't think the provider .get was updated to use PG?

Reply:
Sure, but where does the consumer care about the provider, so long as the eventObject passes validation?